Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundlePagePrivate.h

```diff
@@ -129,7 +129,6 @@ WK_EXPORT void WKBundlePageClearApplicationCache(WKBundlePageRef page);
 WK_EXPORT void WKBundlePageClearApplicationCacheForOrigin(WKBundlePageRef page, WKStringRef origin);
 WK_EXPORT void WKBundlePageSetAppCacheMaximumSize(WKBundlePageRef page, uint64_t size);
 WK_EXPORT uint64_t WKBundlePageGetAppCacheUsageForOrigin(WKBundlePageRef page, WKStringRef origin);
-WK_EXPORT void WKBundlePageSetApplicationCacheOriginQuota(WKBundlePageRef page, WKStringRef origin, uint64_t bytes);
 WK_EXPORT void WKBundlePageResetApplicationCacheOriginQuota(WKBundlePageRef page, WKStringRef origin);
```

Comment:
Why not deprecate or remove this?

Reply:
I remove the ones that are not in use. Some of them are still in use by WebKitTestRunner, I will modify tests and remove the rest in followup.